src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -208,9 +215,22 @@ private void AddSecrets(TransactionBuildContext context)
                 return;
 
             Wallet wallet = this.walletManager.GetWalletByName(context.AccountReference.WalletName);
-
+            Key privateKey; 
+            SHA256 sha256 = SHA256.Create();
             // get extended private key
-            var privateKey = Key.Parse(wallet.EncryptedSeed, context.WalletPassword, wallet.Network);
+            string cacheKey = Encoding.Default.GetString(sha256.ComputeHash(Encoding.UTF8.GetBytes(wallet.EncryptedSeed)));
```

Comment:
Why is the cacheKey created in his way? It's just an index for the seed in the MemoryCache, right? (I spent 5 minutes trying to understand why it's hashed, encoded etc.)
I prefer the simple approach of doing `string cacheKey = wallet.EncryptedSeed;`.
Would that work?